docusaurus/docs/how-to-guides/data-source-plugins/error-handling-in-data-source-plugins.md

```diff
@@ -58,20 +56,6 @@ Grafana displays the exception message in the UI as written, so use grammaticall
 
 :::
 
-## Common error scenarios
-
-Here are some examples of situations where you might want to display an error to the user.
-
-### Invalid query response
+## See also
 
-Users have full freedom when they create data source queries for panels. If your panel plugin requires a specific format for the query response, then use the panel canvas to guide the user.
-
-```ts
-if (!numberField) {
-  throw new Error('Query result is missing a number field');
-}
-
-if (frame.length === 0) {
-  throw new Error('Query returned an empty result');
-}
-```
+- [Error handling in panel plugins](../panel-plugins/error-handling-for-panel-plugins.md)
```

Comment:
Added this as we have a redirect from the previous, plugin-type agnostic page to here, which is fine as a catch instead of a broken link. But best to include pointers to other topics. Will also update with app when merged